.github/workflows/build.yaml

```diff
@@ -133,7 +133,7 @@ jobs:
         with:
           sarif_file: "trivy-results.sarif"
 
-  integration-test:
+  integration-test: # caching tests are disabled due to slow file I/O in github actions
```

Comment:
Kinda funny that a cache test is impacted by slow I/O 😆

Reply:
:sob:

Reply:
Time for me to find a new career